common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -470,6 +495,16 @@ open class MusicService : MediaLibraryService() {
 
                 return Futures.immediateFuture(SessionResult(SessionResult.RESULT_SUCCESS))
             }
+
+            if (customCommand.customAction == SEEK_BACK) {
+                exoPlayer.seekBack()
```

Comment:
These commands are replicating functionality that's already handled by standard commands defined by a `Player`, which we don't want to encourage (see the note at the end of [this section](https://developer.android.com/guide/topics/media/session/mediasession#adding-custom)) - could you update these commands to do something different? Here are some ideas:

- Jump to a random item in the current playlist (see one of my previous comments for some pointers on how to execute on this)
- Copy the current song title to the clipboard (https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Copying)
- Add the current song to some sort of "favorites" list (You could do similar logic to what we have [here](https://github.com/android/uamp/blob/compose/common/src/main/java/com/example/android/uamp/media/PersistentStorage.kt), but with `FAVORITE_SONG_*_KEY`)